web/web/auth/static/reducer.js

```diff
@@ -1,15 +1,16 @@
 const initialState = {
     email: "",
-    token: null
+    userLoggedIn: false
 }
 
 export default function auth(state = initialState, action) {
     switch (action.type) {        
         case "LOGIN_SUCCESSFUL":
-            return { ...state, token: action.token }
-
+            return { ...state, userLoggedIn: action.userLoggedIn }
         case "LOGIN_FAILED":
-            return { ...state, token: null }
+            return { ...state, userLoggedIn: action.userLoggedIn }
+        case "RESET_USER_LOGGED_IN":
+            return { ...state, userLoggedIn: action.userLoggedIn }
```

Comment:
Since these all do the same, you can consolidate it into:
```javascript
case "LOGIN_SUCCESSFUL":
case "LOGIN_FAILED":
case "RESET_USER_LOGGED_IN":
    return { ...state, userLoggedIn: action.userLoggedIn }
```

Reply:
awesome, thanks